content/chat/rooms/occupancy.textile

```diff
@@ -0,0 +1,99 @@
+
+The following is an example payload of an occupancy event:
+
+```[json]
+{
+  "connections": 103;
+  "presenceMembers": 95;
+}
+```
+
+The following are the properties of an occupancy event payload:
+
+|_. Property |_. Description |_. Type |
+| connections | The number of connections in the room. | Number |
+| presenceMembers | The number of users entered into the "presence set":/chat/rooms/presence of the room. | Number |
+
+Two different metrics are returned so that you can differentiate between anonymous and identified users. A user must be identified with a @clientId@ in order to register their "presence":/chat/rooms/presence in a room. Therefore @presenceMembers@ is a count of identified users in the room, and @connections@ returns the number of anonymous and identified users in the room.
+
+<aside data-type='note'>
+<p>A user is counted for every device that they are in the room with. For example, if a user enters a room on their phone and their desktop, this counts as two connections. Similarly if they enter into presence for the room on both devices, this also counts as two unique presence members.</p>
+</aside>
+
+h3(#unsubscribe). Unsubscribe from room occupancy
+
+Unsubscribe from room occupancy to remove previously registered listeners.
+
+Use the provided @unsubscribe()@ function to remove a listener's subscription:
+
+```[javascript]
+// Initial subscription
+const { unsubscribe } = room.occupancy.subscribe((event) => {
+  console.log(event);
+});
+
+// To remove the listener
+unsubscribe();
+```
+
+Use the "@occupancy.unsubscribeAll()@":https://sdk.ably.com/builds/ably-labs/ably-chat-js/main/typedoc/interfaces/Occupancy.html#unsubscribeAll method to remove all occupancy listeners in a room:
+
+```[javascript]
+await room.occupancy.unsubscribeAll();
+```
+
+h3(#retrieve). Retrieve room occupancy
+
+The occupancy of a room can be retrieved in one-off calls instead of subscribing to updates.
+
+Use the "@occupancy.get()@":https://sdk.ably.com/builds/ably-labs/ably-chat-js/main/typedoc/interfaces/Occupancy.html#get method to retrieve the occupancy of a room:
+
+```[javascript]
+const occupancy = await room.occupancy.get();
+```
+
+The following is an example payload of an occupancy event:
+
+```[json]
+{
+  "connections": 103;
+  "presenceMembers": 95;
+}
+```
+
+The following are the properties of an occupancy event payload:
+
+|_. Property |_. Description |_. Type |
+| connections | The number of connections in the room. | Number |
+| presenceMembers | The number of users entered into the "presence set":/chat/rooms/presence of the room. | Number |
+
+Two different metrics are returned so that you can differentiate between anonymous and identified users. A user must be identified with a @clientId@ in order to register their "presence":/chat/rooms/presence in a room. Therefore @presenceMembers@ is a count of identified users in the room, and @connections@ returns the number of anonymous and identified users in the room.
```

Comment:
Same as the comment above. Separately, I'm not sure if we need to repeat this entire explanation as well as payload props?